javascript/ql/src/semmle/javascript/security/dataflow/CodeInjectionNodes.qll

```diff
@@ -0,0 +1,102 @@
+/**
+ * Provides data flow node interfaces and default implementations for reasoning about code injection.
```

Comment:
Please rephrase this comment to be more descriptive. How about saying that this module contains the default sources, sinks and sanitisers for code injection, as well as extension points for adding your own?